android/app/src/main/AndroidManifest.xml

```diff
@@ -1,25 +1,27 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-  package="com.appwithsignin">
+          package="com.appwithsignin">
```

Comment:
what is that changes mean? Please every time check your PR from the top to bottom. I suppose that is not necessary changes

Reply:
I'm sorry, I'll be more careful